src/services/auth/index.tsx

```diff
@@ -172,5 +191,97 @@ export const mockValidateOTP = async (
   };
 };
 
+/** Error message returned by endpoints without a /auth/logout handler */
+const oldBackendError =
+  "Authorization header requires 'Credential' parameter. Authorization header requires 'Signature' parameter. Authorization header requires 'SignedHeaders' parameter. Authorization header requires existence of either a 'X-Amz-Date' or a 'Date' header. Authorization=";
+
+export const liveCallLogout = async (
+  sessionToken: string,
+  operatorToken: string,
+  endpoint: string
+): Promise<void> => {
+  const payload = { sessionToken, operatorToken };
+
+  try {
+    await fetchWithValidator(LogoutResponse, `${endpoint}/auth/logout`, {
+      method: "POST",
+      headers: {
+        Authorization: sessionToken,
+        "Content-Type": "application/json",
```

Comment:
Note for future reference:
Required this line here as without this header the `fetch` call would throw a `TypeError: Network service unavailable` without even attempting to call the backend. No packet would be sent out.